CMSgov/eAPD
# Tbolt/2830 checks for error before displaying Saved! message in header

Resolves #2830 

### This pull request changes...

- When there is an error saving the page, it will no longer show the "Saved!" message in the header. Instead it will use "Last saved" time.

### Testing guidance
1. First login to staging and verify current behavior by completing steps 3-6. You should see that even when Offline it is showing a "Saved!" message in the header.
2. Login to PR environment
3. Create/view APD
4. Make some changes
5. Use dev tools to set the network to Offline
6. Make more edits
7. Header should **not** say "Saved!" instead it should display the last saved time

### This pull request is ready to merge when...

- [x] Develop has updated automated tests (and all tests are passing)
- [ ] QA has manually tested and created bugs for any issues found (beyond minor fixes)

### This feature is done when...
